pontoon/sync/tests/formats/test_ftl.py

```diff
@@ -0,0 +1,111 @@
+import os
+import shutil
+import tempfile
+
+from textwrap import dedent
+
+from django_nose.tools import (
+    assert_equal,
+    assert_raises,
+    assert_true,
+)
+from pontoon.base.tests import (
+    TestCase,
+    create_named_tempfile,
+    LocaleFactory,
+)
+from pontoon.sync.formats import ftl
+from pontoon.sync.tests.formats import FormatTestsMixin
+
+
+class FTLResourceTests(FormatTestsMixin, TestCase):
+    def setUp(self):
+        super(FTLResourceTests, self).setUp()
+        self.tempdir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        super(FTLResourceTests, self).tearDown()
+        shutil.rmtree(self.tempdir)
+
+    def get_nonexistant_file_resource(self, path):
+        contents = dedent("""text = Arise,awake and do not stop until the goal is reached.""")
```

Comment:
Please add space after comma.

Reply:
As, [__init__](https://github.com/mozilla/pontoon/blob/master/pontoon/sync/tests/formats/__init__.py#L363) i think this is getting the `key` attribute from the [test_compare_locale](https://github.com/mozilla/pontoon/blob/master/pontoon/sync/tests/formats/test_compare_locales.py#L241) `name` property of this XML

But when i am writing test for the FTL format [test_ftl](https://github.com/aniruddha2000/pontoon/blob/bug-1521820/pontoon/sync/tests/formats/test_ftl.py#L226) the FTL variable name is `Missing-String` but in [__init__](https://github.com/mozilla/pontoon/blob/master/pontoon/sync/tests/formats/__init__.py#L363) case the key is getting from `Missing String` so the `missing_translation` variable will not valid and [this-line](https://github.com/mozilla/pontoon/blob/master/pontoon/sync/tests/formats/__init__.py#L364) will show an error `AttributeError: 'NoneType' object has no attribute 'strings'` 

If i change [this](https://github.com/mozilla/pontoon/blob/master/pontoon/sync/tests/formats/__init__.py#L363) `key` attribute to `Missing-String` then `test_compare_locale` will show an error `AttributeError: 'NoneType' object has no attribute 'strings'` and i also can't give a space in between the FTL variable.
That's I am facing the main problem.